### PR TITLE
Morbidity function

### LINKED ISSUE #45: Morbidity
Add morbidity function for net benefit/cost function (technology.py)

Depends on #46 and #47

### PR DESCRIPTION
**Pull request by babakkhavari**
reads tech and the social specs file, add morbidity and mortaltiy to  each technology. This fixes #45

### CODE REVIEW

File: onsstove/technology.py [hunk 1]
```diff
@@ -86,206 +86,74 @@ def relative_risk(self):
...
 
-    i = 1
-    morb_U_vector = []
-    morb_R_vector = []
-    while i < 6:
-        morbidity_alri_U = cl_alri[i] * coi_alri * morb_alri_U / (1 + discount_rate_social) ** (end_year - start_year)
-        morbidity_copd_U = cl_copd[i] * coi_copd * morb_copd_U / (1 + discount_rate_social) ** (end_year - start_year)
-        morbidity_lc_U = cl_lc[i] * coi_lc * morb_lc_U / (1 + discount_rate_social) ** (end_year - start_year)
-        morbidity_ihd_U = cl_ihd[i] * coi_ihd * morb_ihd_U / (1 + discount_rate_social) ** (end_year - start_year)
+        rr_alri, rr_copd, rr_ihd, rr_lc = relative_risk(self)
```
Comment (camiloramirezgo) [inline onsstove/technology.py:107]: In here the call to the method should be `self.relative_risk()`

File: onsstove/technology.py [hunk 2]
```diff
@@ -86,206 +86,74 @@ def relative_risk(self):
...
 
-        morb_U_total = morbidity_alri_U + morbidity_copd_U + morbidity_lc_U + morbidity_ihd_U
+        paf_alri = paf(rr_alri, sfu)
+        paf_copd = paf(rr_copd, sfu)
+        paf_ihd = paf(rr_ihd, sfu)
+        paf_lc = paf(rr_lc, sfu)
```
Comment (camiloramirezgo) [inline onsstove/technology.py:112]: As `paf` is now a class method, then it also needs to be called as `self.paf(agr1, agr2)`

**Review by camiloramirezgo — CHANGES_REQUESTED** [review]

File: onsstove/technology.py (hunk 2 above)
Comment (babakkhavari) [reply onsstove/technology.py]: Fixed

File: onsstove/technology.py (hunk 1 above)
Comment (babakkhavari) [reply onsstove/technology.py]: Fixed

**Review by camiloramirezgo — APPROVED** [review]
Looks nice!

**Pull request merged**